mailbag/formats/mbox.py

```diff
@@ -37,15 +37,28 @@ def messages(self):
             data = mailbox.mbox(filePath)
             for mail in data.itervalues():
                 try:
-                    mailObject = email.message_from_bytes(mail.as_bytes())
+                    mailObject = email.message_from_bytes(mail.as_bytes(),policy=email.policy.default)
                     print (dir(mail))
                     # Try to parse content
+                    attachmentNames = []
+                    attachments = []
+                    
+                    body = mailObject.get_body(preferencelist=('related', 'html', 'plain')).__str__()
+                    
+                    # Extract Attachments
+                    for attached in mailObject.iter_attachments():
+                        attachmentName,attachment = helper.saveAttachments(attached)
+                        if attachmentName:
+                            attachmentNames.append(attachmentName)
+                            attachments.append(attachment)
                     if mail.is_multipart():
```

Comment:
This needs to be switched over to use `.get_body()` and `.iter_attachments` just like the EML parser.

Reply:
Your already using `.iter_attachments` so just `,get_body()` instead of `.walk()`